common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1238,7 +1238,11 @@ def submit_problem(self, data, override_time=False):
             # without a stack trace
             else:
                 # Translators: {msg} will be replaced with a problem's error message.
-                msg = _(u"Error: {msg}").format(msg=inst.message)
+                if type(inst) == StudentInputError:
+                    msg = _(u"{msg}").format(msg=inst.message)
+                else:
+                    msg = _(u"Error: {msg}").format(msg=inst.message)
```

Comment:
Do we have to remove the "Error" prefix? StudentInputError is used for a LOT of different return messages, and I don't know if we really want to remove "Error" from all of them.